### ISSUE TITLE
Request: Support for read+write variable pairs

### ISSUE THREAD

**Issue by Asmageddon** [markusfisch/ShaderEditor#139]
Even if it were to be something unwieldy like separate `buffer0_in` and `buffer0_out` variables, I would very, very much so like to request that. For both textures and other data types.

**Comment by AntonPieper** [comment]
What exactly do you mean by that?

Something like this?
```glsl
uniform sampler2D backbuffer; // [bind = fragColor]
uniform sampler2D backbuffer2; // [bind = somethingElse]

out vec4 fragColor; // [output]
out vec2 somethingElse;
```

Or are you asking for a feature like ShaderToy's passes?

**Comment by Asmageddon** [comment]
Yes, exactly. Something that allows the transfer of information from one frame to the next. 

Multiple passes would also be great to have, but that's not my current request.
